### ISSUE TITLE
Configurable Option To Toggle Whether Payout From the vTrade Action Payout Is Multiplied By The Amount Of Trade Output

### ISSUE THREAD

**Issue by DC-FLASH** [Zrips/Jobs#571]
**Description of new idea or improvement on existing one**
Basically when I was testing the vTrade action on 1.14.4 I found that when trading for items when the output is more than one item, for example trading an emerald for 4 stone, instead of being paid for a single stone you get paid for all 4, which can make sense in all honesty and if it is an issue I suppose you can just reduce the payout for it but if it isn't an issue a option to toggle between a singular payout, similar to the option for the crafting job where you can either craft multiple things in one go and get paid for all or just one.

**Detailed information what should do what and when**
Basically an option to make it so trading for something that gives more than 1 item payout out for the singular item instead of multiplying the payout by the amount of item you get out from the trade similar to option in crafting
